modules/op-tkn-pipeline-management.adoc

```diff
@@ -16,45 +16,45 @@ $ tkn pipeline --help
 
 == pipeline delete
 
-Delete a Pipeline.
+Delete a pipeline.
 
-.Example: Delete the `mypipeline` Pipeline from a namespace
+.Example: Delete the `mypipeline` pipeline from a namespace
 [source,terminal]
 ----
 $ tkn pipeline delete mypipeline -n myspace
 ----
 
 == pipeline describe
-Describe a Pipeline.
+Describe a pipeline.
 
-.Example: Describe `mypipeline` Pipeline
+.Example: Describe `mypipeline` pipeline
 [source,terminal]
 ----
 $ tkn pipeline describe mypipeline
 ----
 
 == pipeline list
-List Pipelines.
+List pipelines.
 
-.Example: Display a list of Pipelines
+.Example: Display a list of pipelines
 [source,terminal]
 -----
 $ tkn pipeline list
 -----
 
 == pipeline logs
-Display Pipeline logs for a specific Pipeline.
+Display pipeline logs for a specific pipeline.
 
-.Example: Stream live logs for the `mypipeline` Pipeline
+.Example: Stream live logs for the `mypipeline` pipeline
```

Comment:
```suggestion
.Example: Stream the live logs for the `mypipeline` pipeline
```
This example changes the verb from "display" to "stream." If stream means "to display a continuously updating list of log messages," I would update line 46 to something like "Display a stream of log messages for a specific pipeline."